src/components/common.types.d.ts

```diff
@@ -175,11 +175,6 @@ interface Instance {
   count: number;
 }
 
-interface InstancesState {
-  running: Instance[];
-  deploying: Instance[];
-}
-
```

Comment:
**❔ question:** actually, should we remove all unused interface in this PR? Including the one related to #919?

Reply:
I can't remove it in this PR because it is used by Application type which needs to be removed in #919 .
We need to merge #919 and when rebasing on master, I will remove this interface.
Thanks for the reminder.